src/Templates/DiffPlaces/createAllBadOpcodeTests.sh

```diff
@@ -103,28 +100,28 @@
 ./createBadOpcodeTest.sh DE
 ./createBadOpcodeTest.sh DF
 
-./createBadOpcodeTest.sh E0
-./createBadOpcodeTest.sh E1
-./createBadOpcodeTest.sh E2
-./createBadOpcodeTest.sh E3
-./createBadOpcodeTest.sh E4
-./createBadOpcodeTest.sh E5
-./createBadOpcodeTest.sh E6
-./createBadOpcodeTest.sh E7
-./createBadOpcodeTest.sh E8
-./createBadOpcodeTest.sh E9
-./createBadOpcodeTest.sh EA
-./createBadOpcodeTest.sh EB
-./createBadOpcodeTest.sh EC
-./createBadOpcodeTest.sh ED
+./createBadOpcodeTest.sh E0 '<Cancun'
```

Comment:
E0 - ED  on cancun are not bad?

Reply:
Yes, they are used for opcodes that have an immediate operand. See https://hackmd.io/@shemnon/CancunOpcodeAllocation .